internal/resources/grafana/resource_user_test.go

```diff
@@ -96,6 +108,16 @@ resource "grafana_user" "test" {
 }
 `
 
+const testAccUserConfig_mixedCase = `
```

Comment:
could potentially add a case for all uppercase

only for all coverage

Reply:
Oh, sorry, didn't see this comment before merging. I think in this case it doesn't matter enough to raise another PR just for that.